src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlSerializer.cs

```diff
@@ -107,7 +107,6 @@ protected virtual SamlAction ReadAction(XmlDictionaryReader reader)
 
                 var action = reader.ReadElementContentAsString();
                 reader.MoveToContent();
-                reader.ReadEndElement();
                 return new SamlAction(action, new Uri(namespaceValue));
```

Comment:
I don't see the need for MoveToContent().
Would return new SamlAction(reader.ReadElementContentAsSeting(), new Uri(namespaceValue)) work?